ggu/main.c

```diff
@@ -100,23 +108,130 @@ process_single_policy(const struct ggu_policy *policy, const struct ggu_config *
 }
 
 static void
-process_single_packet(struct rte_mbuf *pkt, const struct ggu_config *ggu_conf)
+ggu_policy_iterator(struct ggu_decision *ggu_decision,
+	unsigned int decision_list_len, ggu_policy_fn policy_fn,
+	void *policy_arg, const char *block)
+{
+	while (decision_list_len >= sizeof(*ggu_decision)) {
+		struct ggu_policy policy;
+		uint8_t decision_type = ggu_decision->type;
+		size_t decision_len;
+
+		switch (decision_type) {
```

Comment:
Let's check that the reserved fields of `ggu_decision` are zero.

Reply:
You forgot to write the verification.

Reply:
It's in the patch, GitHub is just not recognizing the changes as directly addressing these comments.

Reply:
Somehow the verification of the reserved fields is in the second patch instead of the first.

Reply:
GitHub still doesn't recognize the change, but it should be in the first patch now.

Reply:
It's showing in the first patch for me now. Thank you.